### ISSUE TITLE
Make CDN

### ISSUE THREAD

**Issue by quitequinn** [anicollection/anicollection.github.io#5]
I basically want this to be the fontawsome of animations

**Comment by darielnoel** [comment]
I totally agree Quinn. But unfortunately we need time and money for this noble cause. I tried to find some alternatives, but seem not working. 
- https://pledgie.com/campaigns/28679
- https://www.kickstarter.com/projects/888703031/anijs-studio

The people only wants use a free product. They don't think about the time spending to create, enhance and fix the project. Sometimes think that the people like that the big marketings agencies fool them. I don't care the money or be famous, I just want to create tools that helps people.

There 3 good tools made by me here:

http://anijs.github.io/

http://anijs.github.io/studio/

http://anicollection.github.io/

The people thinks that are cool!!!, but just a few people helps us with code or money. You are one of them :)

Well my friend, I would be gratefull if you will share your opinion with me. Thanks in advance.

**Comment by quitequinn** [comment]
I don't know what to say Dariel. Maybe there are opensource grants you could apply for.
https://stripe.com/blog/stripe-open-source-retreat
https://www.mozilla.org/en-US/grants/
http://foundation.travis-ci.org/grants/
http://techfund.comcast.com/

Otherwise just do it like other OS projects, with your free time. Or don't do it.

with <3
